nemo/collections/asr/__init__.py

```diff
@@ -23,3 +23,10 @@
 
 # Set collection name.
 __description__ = "Automatic Speech Recognition collection"
+
```

Comment:
Revert ?

Reply:
This should be reverted automatically by black :)
The action is not working now since it has no custom github token